httpx/_dispatch/wsgi.py

```diff
@@ -7,6 +8,19 @@
 from .base import SyncDispatcher
 
 
+def _get_non_empty_chunk(body: typing.Iterable) -> typing.Iterable:
+    """
+    Get a non-empty chunk from body. This is needed because the status returned
+    by start_response shouldn't be used until the first non-empty chunk has been
+    served.
```

Comment:
Nit: can we move this comment to when we actually call this function? (What this function does is quite clear, but it might be better to keep the motivation close to the actual usage.)

Reply:
Sure.  The style where I work is to avoid comments, so it's just a habit.